packages/hardhat-core/package.json

```diff
@@ -55,40 +55,40 @@
   ],
   "devDependencies": {
     "@nomiclabs/eslint-plugin": "^1.0.0",
-    "@types/chai": "^4.2.0",
-    "@types/chai-as-promised": "^7.1.3",
-    "@types/ci-info": "^2.0.0",
-    "@types/debug": "^4.1.4",
-    "@types/find-up": "^2.1.1",
-    "@types/fs-extra": "^5.1.0",
-    "@types/glob": "^7.1.1",
-    "@types/lodash": "^4.14.123",
-    "@types/mocha": "^5.2.6",
-    "@types/node": "^10.17.24",
-    "@types/node-fetch": "^2.3.7",
-    "@types/qs": "^6.5.3",
-    "@types/resolve": "^1.17.1",
-    "@types/semver": "^6.0.2",
-    "@types/sinon": "^9.0.8",
-    "@types/uuid": "^3.4.5",
-    "@types/ws": "^7.2.1",
-    "@typescript-eslint/eslint-plugin": "^4.28.0",
-    "@typescript-eslint/parser": "^4.28.0",
-    "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
-    "eslint": "^7.29.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.23.4",
-    "eslint-plugin-prettier": "^3.4.0",
-    "ethers": "^5.0.0",
-    "mocha": "^7.1.2",
+    "@types/chai": "4.2.21",
+    "@types/chai-as-promised": "7.1.4",
+    "@types/ci-info": "2.0.0",
+    "@types/debug": "4.1.7",
+    "@types/find-up": "2.1.1",
+    "@types/fs-extra": "5.1.0",
+    "@types/glob": "7.1.4",
+    "@types/lodash": "4.14.172",
+    "@types/mocha": "5.2.7",
+    "@types/node": "12.20.19",
```

Comment:
@alcuadrado can you double-check that this is the version we want to use here?

Reply:
double-checked ✅